### PR TITLE
F-45I Add/Remove cap tests

### PR DESCRIPTION
**Pull request by swapnil-khandare**
Signed-off-by: Swapnil Khandare <swapnil.khandare@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [x] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by swapnil-khandare** [comment]
PASSED tests/csm/rest/test_iam_users.py::TestIamUserRGW::test_39029
PASSED tests/csm/rest/test_iam_users.py::TestIamUserRGW::test_39030
PASSED tests/csm/rest/test_iam_users.py::TestIamUserRGW::test_39031
PASSED tests/csm/rest/test_iam_users.py::TestIamUserRGW::test_39217

File: tests/csm/rest/test_iam_users.py [hunk 1]
```diff
@@ -1932,3 +1932,185 @@ def test_38094(self):
...
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-39029')
+    def test_39029(self):
+        """
+        Add-remove random capabilities
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("[START] Creating IAM user")
```
Comment (aditipande96) [inline tests/csm/rest/test_iam_users.py:1947]: Can you just add log messages of steps present in Test Design for better understanding of test functionality?

File: tests/csm/rest/test_iam_users.py [hunk 2]
```diff
@@ -1932,3 +1932,185 @@ def test_38094(self):
...
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-39030')
+    def test_39030(self):
+        """
+        Test add/remove capabilities with tenant
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("[START] Creating IAM user")
```
Comment (aditipande96) [inline tests/csm/rest/test_iam_users.py:1993]: Above comment applicable to all tests.

File: tests/csm/rest/test_iam_users.py (hunk 1 above)
Comment (swapnil-khandare) [reply tests/csm/rest/test_iam_users.py]: Added

File: tests/csm/rest/test_iam_users.py (hunk 2 above)
Comment (swapnil-khandare) [reply tests/csm/rest/test_iam_users.py]: Added

**Review by kachhwahadivya — APPROVED** [review]

**Review by aditipande96 — APPROVED** [review]

**Review by sonalk0209 — APPROVED** [review]
Approving for merge!!

**Review by ketanarlulkar — APPROVED** [review]

**Pull request merged**